src/components/call_out/_call_out.scss

```diff
@@ -27,6 +28,12 @@
     @include ouiCallOutTitle;
     margin-bottom: 0; // In case it's nested inside OuiText
   }
+
+  .ouiCallOut__closeIcon {
+    position: absolute;
+    right: 6px;
```

Comment:
I feel like this should use `$ouiSize` in some way. Maybe `calc($ouiSizeM / 2)`?

Reply:
Changed to `$ouiSizeS / 2`